samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,351 @@
+
+.. warning::
+   Every time a sensor sample is sent, it will interrupt any attempted GNSS fix.
+   If you set this interval too short, there will not be enough time to ever acquire a fix.
+   In good conditions, and with AGPS data, one minute is generally just barely long enough to get a fix most of the time.
+
+.. _CONFIG_GNSS_ANTENNA_ONBOARD:
+
+CONFIG_GNSS_ANTENNA_ONBOARD - Select the onboard GNSS antenna (default)
+   This option enables the onboard GNSS antenna.
+
+
+.. _CONFIG_GNSS_ANTENNA_EXTERNAL:
+
+CONFIG_GNSS_ANTENNA_EXTERNAL - Select an external GNSS antenna
+   This option enables the external GNSS antenna.
+
+
+.._CONFIG_GNSS_AT_MAGPIO:
+
+CONFIG_GNSS_AT_MAGPIO - ``AT%XMAGPIO`` command
+   Sets the ``AT%XMAGPIO`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
+
+.._CONFIG_GNSS_AT_COEX0:
+
+CONFIG_GNSS_AT_COEX0 - ``AT%XCOEX0`` command
+   Sets the ``AT%XCOEX0`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
+
+.._CONFIG_GNSS_FIX_TIMEOUT_SECONDS:
+
+CONFIG_GNSS_FIX_TIMEOUT_SECONDS - GNSS Fix timeout (seconds)
+   On each location sample, try for this long to achieve a GNSS fix before falling back to cellular positioning.
+
+.. warning::
+	Set this to 20 or so seconds below the sensor sample interval so that there is time left over to perform cellular positioning.
+	Otherwise, location samples may occur at a longer interval than requested.
+
+.. _CONFIG_LOCATION_SAMPLE_INTERVAL_SECONDS:
+
+CONFIG_LOCATION_SAMPLE_INTERVAL_SECONDS - Location sampling interval (seconds)
+   Sets the location sampling interval in seconds.
+
+.. warning::
+   The default sensor sampling interval, 60 seconds, will quickly consume cellular data, and should not be used in a production environment.
+   Instead, consider using a less frequent interval, and, if necessary, combining multiple sensor samples into a single device message, or combining multiple `device messages <nRF Cloud Device Messages_>`_ using the `d2c/bulk device message topic <nRF Cloud MQTT Topics_>`_.
+   Bear in mind that if you significantly increase the sampling interval, you must keep :kconfig:option:`CONFIG_MQTT_KEEPALIVE` short to avoid the carrier silently closing the MQTT connection due to inactivity, which could result in dropped device messages.
+   The exact maximum value depends on your carrier. In general, a few minutes or less should work well.
+
+.. warning::
+   Note also that decreasing the sensor sampling interval too much can prevent GNSS lock.
+   This is because the modem can only receive GNSS data when it is not transmitting on the LTE network.
+   Sample intervals of less than about one minute will prevent the modem from being idle long enough to obtain a GNSS lock.
```

Comment:
```suggestion
   Note also that decreasing the sensor sampling interval too much leads to more frequent use of LTE connection, which can prevent GNSS from obtaining a fix.
   This is because GNSS can operate only when the LTE connection is not active.
   The exact time required to obtain a GNSS fix will vary depending on e.g. the current visibility of satellites, time since last fix and the type of the antenna used.
```

Reply:
These statements were in the wrong section (I think -- either that, or it as the other set of statements). I have merged both duplicated statement sets into a single statement set in the correct location.